prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -0,0 +1,92 @@
+package gov.cdc.prime.router.tokens
+
+import com.google.common.net.HttpHeaders
+import com.microsoft.azure.functions.ExecutionContext
+import com.microsoft.azure.functions.HttpRequestMessage
+import com.microsoft.azure.functions.HttpResponseMessage
+import gov.cdc.prime.router.Organization
+import gov.cdc.prime.router.azure.*
```

Comment:
Our style is to not use .*
Can you change this? Thanks!

Reply:
done!